Purchases/Purchasing/StoreKitAbstractions/SK1StoreProduct.swift

```diff
@@ -15,11 +15,16 @@ import StoreKit
 
 internal struct SK1StoreProduct: StoreProductType {
 
-    init(sk1Product: SK1Product) {
+    init(
+        sk1Product: SK1Product,
+        priceFormatterProvider: PriceFormatterProvider = .init()
```

Comment:
This isn't being overridden, so it doesn't need to be a parameter of the constructor right?

Reply:
yeah, we could initialize the provider when the property is defined 👍🏼